app/js/views/modal.jsx

```diff
@@ -21,3 +21,10 @@ export default class Modal extends React.Component {
     );
   }
 }
+
+Modal.propTypes = {
```

Comment:
I agree. I like the use of `propTypes` because it can be useful to all sort of static analysis tools.
I just hope that won't be too much troubles when creating/changing components.
Also it may add some useless code that's not really needed on the production code (The prop validation only happens on development mode of React). If the `propTypes` property of components are left as is by Babel/React maybe we can write a gulp task to remove them from the transpiled code.